app/main.ipynb

```diff
@@ -32,17 +34,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "db_con = sqlite3.connect(\"../db.sqlite3\")\n",
+    "query = \"\"\"\n",
+    "SELECT datetime, quantity, price, name\n",
+    "FROM restaurant_order\n",
+    "JOIN restaurant_orderitem ON restaurant_order.id = restaurant_orderitem.order_id\n",
+    "JOIN restaurant_product ON restaurant_orderitem.product_id = restaurant_product.id;\n",
```

Comment:
Ensure that the table and column names in the SQL query match those in your database schema. The query assumes specific table structures which might not be correct if the database schema is different.